R/FDAFeatWavelets.R

```diff
@@ -0,0 +1,33 @@
+#' @title Discrete Wavelet transform features.
+#'
+#' @description
+#' The function extracts discrete wavelet transform coefficients from the raw
+#' functional data.
+#'
+#' @param data [\code{data.frame}]\cr
+#'   Functional data.
+#' @param target [\code{character}]\cr
+#'   Name of the target variable. Default: \dQuote{NULL}. The variable is only
+#'   set to be consistent with the API.
+#' @param filter,boundary [\code{character}]\cr
+#'   Specifies which filter or boundary should be used. Default:
+#'   \code{filter} = \dQuote{la8}, \code{boundary} = \dQuote{periodic}. See
+#'   package \code{wavelets} for more information.
```

Comment:
The link did not work in the previous versions (just threw errors) thats why i removed it